### PR TITLE
I18n generators update: use @wix/wix-i18n-config

### PR DESCRIPTION
**Pull request by vybu**
### 🔦 Summary
Use all new [@wix/react-i18n-config](https://github.com/wix-private/fed-infra/tree/master/wix-i18n-config) in all of the generators.
All changes made with as little of a footprint as possible.

Business manager generator was modified to load messages with dynamic imports.
Full stack example was modified to attach messages to templates during server time and use it synchronously during client side initialization, instead of loading them dynamically in client side.

### CODE REVIEW

**Comment by itaibenda** [comment]
@yairhaimo will someone have time to take this soon?

**Comment by itaibenda** [comment]
*bump*
To whom it may concern 🤷🏻‍♂️

**Comment by itaibenda** [comment]
💔

**Comment by itaibenda** [comment]
@ranyitz is there a technical reason why this wasn't merged or it merely went unnoticed?

@asaf-yonay @yaarams @yaelz @gonengar FYI

File: packages/create-yoshi-app/templates/out-of-iframe/javascript/src/components/Widget/Widget.js [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
 import { TPAComponentsProvider } from 'wix-ui-tpa/TPAComponentsConfig';
 import { Button } from 'wix-ui-tpa/Button';
-import i18n from '../../config/i18n';
 import styles from './Widget.st.css';
 
 export default class WidgetRoot extends React.Component {
   render() {
     const { name, locale, experiments, mobile } = this.props;
 
+    const i18n = initI18n({
```
Comment (itaibenda) [inline packages/create-yoshi-app/templates/out-of-iframe/javascript/src/components/Widget/Widget.js]: should be memoized per locale or be called outside of the render function

**Comment by itaibenda** [comment]
@amitdahan TODO:
1. Update `wix-i18n-config` to version 2
2. Load the polyfills for every application/platform type

**Comment by ranyitz** [comment]
#2390 #2392

**Pull request closed without merging**